examples/09_Prediction.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 9. Predicting with Prognostics Models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predicting a specific event"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prediction horizon"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When performing a prediction, it is possible to specify the time value to predict to, or the prediction `horizon`. This prediction horizon marks the end of the \"time of interest\" for the prediction. Often this represents the end of a mission or sufficiently far in the future where the user is unconcerned with the events that occur after this time.\n",
```

Comment:
```suggestion
    "When performing a prediction, it is sometimes desirable to specify a maximum limit for the prediction, or the prediction `horizon`. This prediction horizon marks the end of the \"time of interest\" for the prediction. Often this represents the end of a mission or sufficiently far in the future where the user is unconcerned with the events that occur after this time.\n",
```

Reply:
@teubert what do you think about, "specify a maximum **time** limit"?

Reply:
I thought about that, but I'm concerned that users might think that's a maximum execution time (i.e., timeout) instead of simulation time. But I'm also not happy about my suggestion. It might be ok to say time limit since you provide more detail in the next sentence. What do you think?

Reply:
I think that could face the same confusion. Let's go with your original suggestion and rely on your next sentence to clear up any confusion.